x/interchainqueries/client/cli/query.go

```diff
@@ -112,3 +113,27 @@ func CmdQueryRegisteredQueryResult() *cobra.Command {
 
 	return cmd
 }
+
+func CmdQueryRemoteHeight() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "query-height [connection-id]",
```

Comment:
`query-height` is a bit misleading actually. Let's call it like `query-latest-known-remote-height` or smth? I don't like this long name but you got the idea I hope :)

Reply:
And change it everywhere definitely